src/Avalonia.X11/X11Window.cs

```diff
@@ -416,7 +415,9 @@ ev.ButtonEvent.button switch
                             2 => RawPointerEventType.MiddleButtonDown,
                             3 => RawPointerEventType.RightButtonDown,
                             8 => RawPointerEventType.XButton1Down,
-                            9 => RawPointerEventType.XButton2Down
+                            9 => RawPointerEventType.XButton2Down,
+                            _ => throw new AvaloniaInternalException(
```

Comment:
I'd rather ignore the event completely than crash the app for no reason

Reply:
Ok, I didn't want to change the semantics (it would previously have thrown a `SwitchExpressionException` here) but that makes more sense to me.

Reply:
Actually, there's an `if` statement around the switch which should make sure that all cases are handled:

```
if (ev.ButtonEvent.button < 4 || ev.ButtonEvent.button == 8 || ev.ButtonEvent.button == 9)
```

Given that there's this check, and that this change isn't introducing an exception where there wouldn't be one before - I'd rather not refactor this logic in this PR. If you want to refactor it in a subsequent PR, I'd say that would be the best place for it.